consensus/consensus.go

```diff
@@ -60,6 +61,9 @@ type Config struct {
 
 	// Path is the directory path for the consensus protocol to store information
 	Path string
+
+	// JSONRPCAddr is the address of JSONRPC server running within the node
+	JSONRPCAddr *net.TCPAddr
```

Comment:
I would say just pass a string instead of `net.TCPAddr` object, since we only use its string representation.